pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -312,6 +312,9 @@ func (o *createChangefeedOptions) run(ctx context.Context, cmd *cobra.Command) e
 
 	tables, err := o.apiClient.Changefeeds().VerifyTable(ctx, verifyTableConfig)
 	if err != nil {
+		if strings.Contains(err.Error(), "ErrInvalidIgnoreEventType") {
+			cmd.Println(filter.SupportedEventWarnMessage())
```

Comment:
```suggestion
			cmd.Println(filter.SupportedEventWarnMessage())
```
`SupportedEventWarnMessage' is weird for me.